tests/gh-actions/install_knative.sh

```diff
@@ -5,5 +5,5 @@ set +e
 kustomize build common/knative/knative-serving/overlays/gateways | kubectl apply -f -
 set -e
 kustomize build common/knative/knative-serving/overlays/gateways | kubectl apply -f -
-kustomize build common/knative/knative-eventing/base | kubectl apply -f -
+kustomize build common/istio-1-16/cluster-local-gateway/base | kubectl apply -f -
```

Comment:
local gateway is no longer required with Knative 1.x

Reply:
yes please, let us get rid of knative-local-gateway.

Reply:
We still need the knative local gateway resource but not the istio local gateway deployment. The istio local gateway is merged to the ingress gateway deployment and listens on a different port.